src/PieChart.js

```diff
@@ -22,7 +22,7 @@ class PieChart extends React.Component {
      * Accessor for getting the values plotted on the pie chart.
      * If not provided, just uses the value itself at given index.
      */
-    getValue: CustomPropTypes.getter,
+    slice: CustomPropTypes.getter.isRequired,
```

Comment:
Much needed rename! This name aligns better with our other charts' accessors (x, y). Since this is a change in the API, noting this as a breaking change!

Reply:
Perhaps we can update the description as well
"Accessor for getting the pie slices plotted on the pie chart. If not provided, just uses the data value itself at the given index.""